ee/clickhouse/models/person.py

```diff
@@ -20,9 +20,10 @@
 from ee.kafka_client.topics import KAFKA_PERSON, KAFKA_PERSON_UNIQUE_ID
 from posthog.models.person import Person, PersonDistinctId
 from posthog.models.utils import UUIDT
+from posthog.settings import TEST
 from posthog.utils import is_clickhouse_enabled
 
-if is_clickhouse_enabled():
+if is_clickhouse_enabled() and TEST:
```

Comment:
why do we need this?

Reply:
Tests rely on these hooks for creating person and person_distinct_id. Porting 60+ test files is not urgent either.